virttest/qemu_devices/qdevices.py

```diff
@@ -2039,7 +2037,7 @@ class QBusUnitBus(QDenseBus):
 
     """ Implementation of bus-unit bus (ahci, ide) """
```

Comment:
Need update this doc, since it is not only for 'unit' option now,  then not only ahci and ide.

Reply:
@yanan-fu I would like to keep the description in QSerialBus instead of here, the class QBusUnitBus can not represent SerialBus directly, the inherited relationship is sufficient to explain their relation.

Reply:
@vivianQizhu  My meaning is `bus-unit` is not suitable, as it can be `nr` too.
And for `ahci, ide`,   there also have  `QAHCIBus` and `QIDEBus`.  I think there is not conflicts with `QBusUnitBus`  and `QSerialBus`.   We can add docs for both of them. What do you think ?